### PR TITLE
updated glob link in README

### PR DESCRIPTION
**Pull request by wommy**
old link doesn't work in postcss8, substituted one that does

edit: DOESNT work

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ table).
 If this behavior is not what you want, look at `skipDuplicates` option
 - **If you are looking for glob, or sass like imports (prefixed partials)**,
```
Comment (RyanZim) [inline README.md:32]: Doesn't look like your plugin supports SASS imports.
```suggestion
- **If you are looking for glob imports**,
```

Comment (wommy) [reply README.md:32]: pardon my ignorance, as i havent used Sass in many years, 

https://sass-lang.com/guide#topic-4

i feel like the original wording's intention was to catch the attention of sass migrates that used partials,
thats how i stumbled across this

ill have another edit up in a second

Comment (wommy) [reply README.md]: yeah did you read my newer commit?

i reworded the context

Comment (RyanZim) [reply README.md]: Yes; it still mentions migrating from SASS partials; when your plugin does not support anything like SASS partials.

Comment (wommy) [reply README.md]: oh i thought SASS's  @imports had glob functionality,
ill push another

**Pull request merged**